lib/stacks/eks-blueprint-stack.ts

```diff
@@ -61,12 +61,21 @@ export class EksBlueprint extends cdk.Stack {
 
         const clusterInfo = clusterProvider.createCluster(this, vpc, blueprintProps.version ?? KubernetesVersion.V1_19);
 
+        const postDeploymentSteps = Array<ClusterPostDeploy>();
+
         for (let addOn of (blueprintProps.addOns ?? [])) { // must iterate in the strict order
             addOn.deploy(clusterInfo);
+            const postDeploy : any  = addOn;
```

Comment:
can you use `ClusterPostDeploy` here as opposed to `any`? Saves you the type cast in the if.

Reply:
I can't, because of VSCode / ts safety. Get a warning that I have to cast to any (<unknown>) then to interface. I tried a few options, this one appeared to be the least verbose with no warnings.